.drone.yml

```diff
@@ -1,20 +1,25 @@
-clone:
-  depth: 50
-  tags: true
-  recursive: true
-build:
+pipeline:
+  # Fetch folders from distributed cache
+  sftp_cache_restore:
+    image: plugins/sftp-cache
+    restore: true
+    mount:
+      - /drone/.ivy2
+      - /drone/.coursier-cache
+      - /drone/.sbt
+      - /drone/.git
+
   tests:
-    image: ensime/ensime:v2.x-cache
+    image: scalacenter/scala-extras:1.0
```

Comment:
No, `scala-extras` is not the image meant to be used for the platform, we already have one for that already.
  
This has been named after the fact that it does not only bundle scala and sbt, but contains extra stuff like jekyll, git, and all sort of utils that may be handy to do scripting in sbt.